src/assets/scss/_navbar.scss

```diff
@@ -328,16 +333,18 @@ nav.navbar {
         top: 0;
         right: 100%;
         left: auto;
-        margin-top: 0;
-        margin-right: 0.125rem;
+        // margin-top: 0;
+        // margin-right: 0.125rem;
+        margin: -1px;
```

Comment:
What the reason for this? Why `-1` and not zero?  Add a comment explaining why you've set it to `-1`. And please don't just comment the rules that you think should be removed and remove them instead.